pkg/server/registry/apigroups/acorn/apps/validator.go

```diff
@@ -209,7 +209,8 @@ func (s *Validator) Validate(ctx context.Context, obj runtime.Object) (result fi
 				project,
 				workloadsFromImage,
 				apiv1cfg.WorkloadMemoryDefault,
-				apiv1cfg.WorkloadMemoryMaximum)
+				apiv1cfg.WorkloadMemoryMaximum,
+				apiv1cfg.RequireComputeClass)
```

Comment:
nit: putting this here means this only happens once.
```suggestion
				z.Dereference(apiv1cfg.RequireComputeClass),
			)
```